manager/orchestrator/updater.go

```diff
@@ -224,10 +222,32 @@ func (u *Updater) updateTask(ctx context.Context, service *api.Service, original
 		case e := <-taskUpdates:
 			updated = e.(state.EventUpdateTask).Task
 			if updated.Status.State >= api.TaskStateRunning {
```

Comment:
I think I need to do something here to handle if the task didn't start but actually failed. As failed is more than running it will continue the update. But it should fail and cancel the new task. Especially not stop the old one. It should probably stop the update process.